electric_slide.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.rubygems_version = "1.2.0"
   s.summary = "Automatic Call Distributor for Adhearsion"
 
-  s.add_runtime_dependency 'adhearsion'
+  s.add_runtime_dependency 'adhearsion', ['~> 2.4']
```

Comment:
We should probably try to be a loose as possible with this version, or add `celluloid` as a runtime dependency instead of specifying an adhearsion version.

Reply:
@terryfinn I tried removing the adhearsion version and adding celluloid as a runtime dependency but I kept getting
`NoMethodError:
  undefined method eager_autoload!' for ActiveSupport::Autoload:Module`
when running `bundle exec rake`
I tried using adhearsion v 1.2.6 and that resulted in this error:
````
An error occurred while loading ./spec/electric_slide_spec.rb.
Failure/Error: if Gem::Version.new(Adhearsion::VERSION) < Gem::Version.new('3.0.0')

ArgumentError:
  Malformed version number string Adhearsion::VERSION
````

Reply:
Specifying adhearsion ~> 2.4 and adding celluloid as a runtime dependency leads to bundle exec rake running without issues and doesn't hang during specs

Reply:
The lowest I can get it to is adhearsion >= 2.1